spinoso-string/src/lib.rs

```diff
@@ -256,12 +98,12 @@ impl Hash for String {
         // [3.0.2] > t.hash
         // => 3398383793005079442
         // ```
-        self.buf.hash(hasher);
+        self.inner.as_slice().hash(hasher);
```

Comment:
Was a vec, is now a slice. Has the same effect from what I see. Alternatively, this could be delegated like other impls

Reply:
yup yup. I'd prefer we push this impl and its docs down into `EncodedString`.